lasagne/layers/recurrent.py

```diff
@@ -200,7 +215,7 @@ def step(input_dot_W_n, hid_previous, *args):
                 original_hid_pre,
                 replace=dict(zip(original_params, new_params)))
```

Comment:
Have you checked that strict does not give an error? I would assume that `precompute=True` would give an error because strict is True and the input_to_hidden parameters are not cloned (or what ever its called :)). I think it would be a little messy to fix because args* then depends on the precompute setting.

Can you write a test that checks that?